codespell_lib/data/dictionary.txt

```diff
@@ -9279,6 +9280,7 @@ disignated->designated
 disinguish->distinguish
 disiplined->disciplined
 disired->desired
+diskretization->discretization
```

Comment:
Can you add the sation variant as a separate typo please.

Also the Wikipedia page mentions discretized, so it might be worth adding this typo against that word and maybe also discrete, discretely, discretion etc. Possibly also discuss etc (although grep.app doesn't find examples of the latter).

Reply:
I added some of your suggestions. I think I will make/lookup a list of commonly misspelled words by germans and add it in my holidays.

Reply:
@peternewman i don't understand the travis checks.
f.e. diskretization is for sure not in aspell, since it is wrong. 
or https://en.wiktionary.org/wiki/discretise exists :-)

Reply:
> I added some of your suggestions. I think I will make/lookup a list of commonly misspelled words by germans and add it in my holidays.

Great, the trick there is probably to add a new dictionary called dictionary-de.txt and if necessary move anything in the main dictionary that's a valid German word (like some of my other suggestions) out into a dictionary-en.txt (and make that default). Then ideally make Travis check all that complication too!

> @peternewman i don't understand the travis checks.
> f.e. diskretization is for sure not in aspell, since it is wrong.
> or https://en.wiktionary.org/wiki/discretise exists :-)

See my comments for some further explanation.